### ISSUE TITLE
Compatibility with Xamarin 5

### ISSUE THREAD

**Issue by gil-rodrigues** [enisn/Xamarin.Forms.InputKit#215]
## Question
Is it possible to use the plugin with Xamarin 5? I tried, but without success.

## What I have tried
I tried to integrate in a new project.
- Opened Visual Studio 2019 and selected blank Xamarin.Forms app;
- Added version 3.7.1 via nuget to common project, android and ios;
- Tried to add the line to Android, but gave me the error: 

```csharp
    Plugin.InputKit.Platforms.Droid.Config.Init(this,savedInstanceState);
```

Error	CS0234	The type or namespace name 'Platforms' does not exist in the namespace 'Plugin.InputKit' (are you missing an assembly reference?)	App1.Android	C:\vsprojects\App1\App1\App1\App1.Android\MainActivity.cs	22	Active

- Saw that the sample doesn't have that line, so I tried without it. Also, I added:

```csharp
            CrossCurrentActivity.Current.Init(this, savedInstanceState);
```

- Result:
The projects compile, however, the radiobuttons don't render. Only the checkboxes render correctly.

**Comment by enisn** [comment]
Can I ask which Android version do you target?

Latest version supports Android 9, 10 and 11
if you target a different android version, you'll not able to see that method.

**Comment by gil-rodrigues** [comment]
Hello enisn,

I am targeting Android 11.0 (API Level 30).

We also had a project targeting Android 10.0 with Xamarin 4.8. The method was visible.
When we upgraded to Xamarin 5.0, the method no longer appeared.

Nice work with the kit, by the way.

**Comment by enisn** [comment]
Thanks for reporting it, I'm working on it

**Comment by enisn** [comment]
As a workaround, you can downgrade to 3.7.0 for Android project.

I'll release a fix for that ASAP.

**Comment by enisn** [comment]
The issue is resolved at `v3.7.2`, 

There was missing Android 11 compiler directives (https://github.com/enisn/Xamarin.Forms.InputKit/commit/77c084003e3cc956ea0cbc5f58cd7fcfc6d60415)

Feel free to reopen this issue if problem continues

**Comment by gil-rodrigues** [comment]
@enisn Thank you, you saved my life. I already tested in Android with success.
